pkg/k8sutils/k8sutils.go

```diff
@@ -168,6 +168,23 @@ func GetService(svcName string, svcNS string) (*v1.Service, error) {
 
 }
 
+// DescribeService updates the service status and returns the status part
+func DescribeService(svc *v1.Service) (*v1.ServiceStatus, error) {
+	client, err := GetK8sClient()
+	if err != nil {
+		return nil, err
+	}
+	svc, err = GetService(svc.Name, svc.Namespace)
```

Comment:
by default the get does not return the status resource?

Reply:
For all of these my initial idea was to get the latest status and so the update status. Maybe I can remove the explicit update from this and other objects too.